tools/nvqpp/nvq++.in

```diff
@@ -367,16 +368,19 @@ while [ $# -ne 0 ]; do
 	-v)
 		ECHO=true
 		;;
-	*.o)
+	*.o | *.so | *.bundle)
 		OBJS="${OBJS} $1"
 		;;
 	*.cpp | *.cc)
 		SRCS="${SRCS} $1"
 		;;
+	*.a | *.dylib)	    
+		LINKLIBS="${LINKLIBS} $1"
+		;;
 	*)
-		#FIXME: issue #258
-		#echo "argument $1 is ignored" >&2
+		# Pass any unrecognized options on to the clang++ tool.
 		ARGS="${ARGS} $1"
+		CUDAQ_QUAKE_ARGS="${CUDAQ_QUAKE_ARGS} --Xcudaq $1"
```

Comment:
I wonder if this would cause a problem when there are target-specific arguments, such as `--quantinuum-machine abc`. 
Would it trigger an unknown argument error in clang?

Reply:
Thanks for the reminder, Tien. I forgot to take care of that.